development/init.sh

```
@@ -1,18 +1,36 @@
 #!/bin/bash
 
+cd /home/frappe
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)